scripts/common_replacement

```diff
@@ -34,12 +34,14 @@
 # as those licenses appear in the file LICENSE-OPENSSL.
 
 scenarioFile=$1
-
+count=1
 . /etc/clearwater/config
 ipv4prefix='0x0001'
 hex_address=$(printf '%02X' ${local_ip//./ })
 sed -e "s/SEAGULL_LOCAL_IP_AS_HEX/$ipv4prefix$hex_address/g" \
     -e "s/SCSCF_URI/sip:$sprout_hostname:5054/g" \
     -e "s/HOME_DOMAIN/$home_domain/g" \
-    -e "s/PUBLIC_HOSTNAME/$public_hostname/g" -i $scenarioFile
+    -e "s/PUBLIC_HOSTNAME/$public_hostname/g" \
+    -e "s/SEAGULL_LOCAL_IP/'$local_ip'/g" \
```

Comment:
I'm seeing `<avp name="Origin-Host" value="'10.184.91.221'"> </avp>` in my scenario file (the interesting thing being how many quotes there are around the IP address), so I assume this isn't right. The variety of quotes in this file are just completely mental though, need a bash expert to get to the bottom of it.